app/code/community/Ebanx/Gateway/Model/Adapters/Paymentadapter.php

```diff
@@ -142,12 +142,18 @@ public function transformPerson($person, $data)
 	{
 		$document = $this->helper->getDocumentNumber($data->getOrder(), $data);
 
+		$email = $person->getCustomerEmail() ?: $data->getBillingAddress()->getEmail();
+
+		$name = $person->getCustomerFirstname() && $person->getCustomerLastname()
```

Comment:
What if the user typed the first name but not the last name? Just replace `&&` with `||`